src/main/java/org/broadinstitute/hellbender/engine/FeatureManager.java

```diff
@@ -80,7 +80,7 @@
      * The tool instance containing the FeatureInput argument values that will form the basis of our
      * pool of FeatureDataSources
      */
-    private final CommandLineProgram toolInstance;
+    private transient final CommandLineProgram toolInstance;
```

Comment:
Except for this change -- do you need this? Thought we addressed this in your partitioner branch